source/util/parse_number.h

```diff
@@ -171,6 +171,7 @@ bool ParseNumber(const char* text, T* value_pointer) {
   static_assert(sizeof(T) > 1,
                 "Don't use a single-byte type this parse method");
```

Comment:
The error message is strange.

Maybe
Single-byte types not supported in this function?